nerfstudio/model_components/losses.py

```diff
@@ -485,3 +485,17 @@ def __get_prediction_ssi(self):
         return self.__prediction_ssi
 
     prediction_ssi = property(__get_prediction_ssi)
+
+def tv_loss(grids: TensorType["grids", "feature_dim", "row", "column"]):
```

Comment:
Add you add a test for this function.

Reply:
Where shall I add the tests? create a test_loss.py in tests/model_components?

Reply:
Yes, that sounds good!